haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku. \
+Intended to be a full-featured organizer. "
+HOMEPAGE="https://sourceforge.net/projects/eventual/"
+COPYRIGHT="2011 Alex Hitech"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="846290b16e76a8a4ffbd955057306a18ac96d5e3"
+SOURCE_URI="https://github.com/HaikuArchives/Eventual/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="e0b9844cebb6ed2cf42f93e592fe2e4ce0b801457b31cbac7ff7f5b5d493f76e"
+SOURCE_FILENAME="Eventual-$srcGitRev.tar.gz"
+SOURCE_DIR="Eventual-$srcGitRev"
+PATCHES="eventual-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 ?x86_64"
+
+PROVIDES="
+	eventual = $portVersion
+	app:Eventual = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:sh
+	"
+
+BUILD()
+{
+	sh makelib.sh
+	cd src/Libraries
+	mkdir -p ../lib
+	for LIBDIR in */ ; do
+		LIBDIR=${LIBDIR%?}
+		NAME=lib$LIBDIR.so
+		LIB=$LIBDIR/objects.x86-cc2-release/$NAME
+		cp $LIB ../EventEditor/$NAME
+		cp $LIB ../EventServer/$NAME
+		cp $LIB ../PreferencesPreflet/$NAME
+		cp $LIB ../lib/$NAME
+	done
+	cd ../
+	cd EventEditor && make OBJ_DIR=objects && cd ../
+	cd EventServer && make OBJ_DIR=objects && cd ../
+	cd PreferencesPreflet && make OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/Eventual
+	mkdir -p $appsDir/Eventual/lib
+	cp -r 'src/EventEditor/objects/Eventual Editor' $appsDir/Eventual
+	cp -r 'src/EventServer/objects/Eventual Server' $appsDir/Eventual
+	cp -r 'src/PreferencesPreflet/objects/Eventual Preferences' $appsDir/Eventual
+	cp -r src/lib $appsDir/Eventual
+	addAppDeskbarSymlink $appsDir/Eventual
```

Comment:
Here's a variant which has a small advantage compare to your `INSTALL()`: it hides the lib subdirectory (of the Eventual directory) in the Deskbar menu.
```
INSTALL()
{
	install -d $appsDir/Eventual/lib
	install -t $appsDir/Eventual/lib src/lib/*
	install -t $appsDir/Eventual \
		"src/EventEditor/objects/Eventual Editor" \
		"src/EventServer/objects/Eventual Server" \
		"src/PreferencesPreflet/objects/Eventual Preferences"

	install -d $dataDir/deskbar/menu/Applications/Eventual
	addAppDeskbarSymlink "$appsDir/Eventual/Eventual Editor" Eventual
	addAppDeskbarSymlink "$appsDir/Eventual/Eventual Server" Eventual
	addAppDeskbarSymlink "$appsDir/Eventual/Eventual Preferences" Eventual
}
```
It has a small downside: we are using addAppDeskbarSymlink in an unusual way (i.e. with a slash

I have an error when I launch any of the Eventual apps:
```
Did not succeed to read the preferences from the file! Error = -2147475440
```

Reply:
Hmm...
I'll make the changes and look into this error... after school :(